backend/routes.js

```diff
@@ -61,6 +61,10 @@ const configureRoutes = ({ oauthApi, elite2Api, keyworkerApi }) => {
   )
   router.use('/api/keyworker-prison-stats', withErrorHandler(controller.getPrisonStats))
 
+  router.use('/manage-key-workers', (req, res, next) => {
+    res.redirect(req.url.replace(/\/manage-key-workers(.*)$/gi, '$1'))
```

Comment:
This is a temporary workaround for other apps pointing to the old URL - it will be removed once they are fixed.